src/components/UpcomingEventsList.tsx

```diff
@@ -33,7 +23,6 @@ const UpcomingEventsList: React.FC = () => {
     Array<IOrderedUpcomingEventType>
```

Comment:
(While we're cleaning up... would switch to `T[]` syntax here over `Array<T>`... Also, we can remove the `: React.FC` typing per conventions)